spec/Store.spec.js

```diff
@@ -85,15 +108,19 @@ describe('Store', () => {
     expect.assertions(1)
     expect(store.modelTypeIndex).toEqual({
       'todos': Todo,
```

Comment:
Do these keys net to be strings?

Reply:
nope! but if we want to enforce this we should do it with a lint. happy to consider in another pr. 👍

Reply:
i went ahead and added a lint, since we were consistent after making this change.

i chose this variant out of personal preference.

https://eslint.org/docs/rules/quote-props#consistent-as-needed

if your preference differs, i'm happy to discuss